extensions/libraries/joomla_entity/src/Fields/Traits/HasFields.php

```diff
@@ -21,11 +22,25 @@
 trait HasFields
 {
 	/**
-	 * Associated fields.
+	 * Associated fields
 	 *
-	 * @var  Collection
+	 * @var  array
```

Comment:
`$fields` is always a Collection because it comes from `$entity->fields()`. Never an array.